### PR TITLE
merge main which includes process changes into v1.1

### PR DESCRIPTION
**Pull request by kdenhartog**
This PR is here to bring main and v1.1 back in alignment. The specific commit I'm looking to bring into V1.1 is the changes to the readme which are defining the process of how we will go about handling changes in the maintenance WG as agreed to via the CCG and merged already in main.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/784.html" title="Last updated on Jul 29, 2021, 4:47 AM UTC (bca9e00)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/784/6c429cf...bca9e00.html" title="Last updated on Jul 29, 2021, 4:47 AM UTC (bca9e00)">Diff</a>

### CODE REVIEW

**Comment by kdenhartog** [comment]
@TallTed this should fix that issue on #783 to separate the two out

File: README.md [hunk 1]
```diff
@@ -29,3 +29,33 @@ mailing list as well.
...
+4. When it's determined a new reccomendation should go out, the W3C Verifiable
+Credentials Working Group members meet, review all the PRs that have been
+merged, and make a formal recommendation if agreement is reached.
+
+### Roadmap for 2021
+- 1 editorial update (v1.1?)
+- 1 substantive update (v1.2?)
+- VC Test Suite Refactoring
+- Start planning VC v2 Work, request a rechartering 3-6 months before end of
+  year to keep VC WG functioning.
```
Comment (TallTed) [inline README.md:55]:
```suggestion
  year to keep VC WG functioning
```

File: README.md [hunk 2]
```diff
@@ -29,3 +29,33 @@ mailing list as well.
...
+
+### Focus areas
+- [v1] Fixing a specific bug
+- [v1] Update examples in the spec to make them modern
+- [v2] VC `@context` needs updating, possibly with security vocab modularized
+  into smaller components instead of all included into a large context file.
```
Comment (TallTed) [inline README.md:61]:
```suggestion
  into smaller components instead of all included into a large context file
```

**Review by brentzundel — APPROVED** [review]

**Comment by kdenhartog** [comment]
@TallTed I agree with your suggested changes, but I'm going to leave them out in this scenario in order to keep alignment between main and V1.1 as close as possible. Main reason being I just want this in so that when commits are targeted at main and I need to update to V1.1 I don't want any wonky git overrides to slow things down. We can merge these separately onto main first and then merge them downstream to this one after if you'd like.

**Pull request merged**